homeassistant/components/mikrotik/__init__.py

```diff
@@ -1 +1,390 @@
 """The mikrotik component."""
+import logging
+import ssl
+import voluptuous as vol
```

Comment:
Please add a blank line between standard library and 3rd party import groups.

Reply:
Adding blank line

Reply:
`ssl` is part of standard library, so the blank line should go between `ssl` and `voluptuous`.